src/main/java/com/exercise/masters/repository/BookRepository.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * 
+ */
+package com.exercise.masters.repository;
+
+import java.util.Optional;
+
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.stereotype.Repository;
+
+import com.exercise.masters.model.Book;
+
+/**
+ * @author michaeldelacruz
+ *
+ */
+
+@Repository
+public interface BookRepository extends PagingAndSortingRepository<Book, Long>{
+	
+	public Optional<Book> findByIdAndAuthorId(Long id, Long authorId);
+	
+	public Optional<Book> findByName(String name);
```

Comment:
This is exact match.
You can use something like below instead:
Page<Book> findByTitleContaining(String title, Pageable pageable)